tests/client-dbus/src/stratisd_client_dbus/_introspect.py

```diff
@@ -266,6 +266,9 @@
     <property name="Uuid" type="s" access="read">
       <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const" />
     </property>
+    <property name="VolumeKeyLoaded" type="v" access="read">
```

Comment:
I need to know precisely what combination of this property being false and the values of other properties should cause stratis-cli to post a warning.

Reply:
I think that sounds correct!

Reply:
Ok! I'm going with, if the pool is encrypted and the volume key is not loaded and it is V2, show a warning.